OneCommunityGlobal/HighestGoodNetworkApp
# Shihao - Lint Fix WeeklySummariesReport

# Description
Lint Fix `src/component/WeeklySummariesReport/**`

## Main changes explained:
- Change the code style according to ESLint and Prettier

## How to test:

1. check into current branch
2. do npm install and ... to run this PR locally
3. log as any user
4. go to dashboard→ summary
5. check the app works as normal
6. set up ESLint and Prettier in VSCode
7. check all files in `src/component/WeeklySummariesReport/**` doesn't have errors
8. Follow the instructions in the video to run the command

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/62367016/b1c7c2c9-8212-4a7e-a4b3-71c09ee1c961
